packages/netsuite-adapter/src/query.ts

```diff
@@ -47,24 +47,40 @@ export type QueryParams = {
   customRecords?: FetchTypeQueryParams[]
 }
 
+export const fullQueryParams: QueryParams = {
```

Comment:
this should be a function, to prevent cases that modifying it in one place will effect other places
```suggestion
export const fullQueryParams = (): QueryParams => ({
```